catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/templates/registryRow.handlebars

```diff
@@ -13,10 +13,10 @@
  --}}
 <td>
     <a href='#{{id}}' class='newLink' data-toggle="modal">
-        {{#if id}}
-            {{id}}
+        {{#if attributes.id}}
```

Comment:
This doesn't look right to me. Attributes is an internal backbone construct, why do you need to call it out in the template? If you're doing a toJSON on the model, it will strip out the internal stuff like `attributes`